### PR TITLE
Feat upgrade search page

### PR DESCRIPTION
**Pull request by haddadsim**
-Change search bar and button to use Material-UI
-Change background color to white

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -13,14 +13,14 @@
 }
 
 .App-header {
-  background-color: #243665;
+  background-color: #ffffff;
```
Comment (george-haddad) [inline src/App.css]: you can shortcut this to just `#fff`

**Comment by haddadsim** [comment]
-Added spinning animation while fetching images from Unsplash.
- Added message if no Images returned.

**Comment by JadTermsani** [comment]
Looks Good 💯

**Review by JadTermsani — APPROVED** [review]

**Review by george-haddad — APPROVED** [review]
Squash and Merge

**Pull request merged**